utils/upload_ansible_roles_to_galaxy.py

```diff
@@ -362,10 +445,20 @@ def locally_clone_and_init_repositories(organization, repo_list):
 def main():
     args = parse_args()
 
-    all_role_whitelist = {"rhel7-role-%s" % p for p in PROFILE_WHITELIST}
+    all_product_whitelist = {"%s" % p for p in PRODUCT_WHITELIST}
+    product_whitelist = set(all_product_whitelist)
+    if args.product:
+        selected_products = {"%s" % p for p in args.product}
```

Comment:
`selected_products = set(args.product)` will do the same job

Reply:
@jan-cerny fixed